github/pull_request.rb

```diff
@@ -179,9 +179,17 @@ def add_status(state, message, options = {})
     @client.create_status(repo.full_name, options[:sha], state, context: 'prprocessor', description: message, target_url: options[:url])
   end
 
+  def get_labels(filename, mapping)
+    mapping.select { |k, v| filename.start_with?(k) }.map { |k, v| v }
```

Comment:
This should be `filename == k || filename.start_with("#{k}/")`. That way you can also match files. You can then remove the normalization code I think.

You can also use `.values` instead of the map.

Reply:
Thanks @ekohl 
Just one problem that I foresee - "directory_label" will be very confusing if we set there files names too. Maybe we should mention this in a document or rename it to "path_label"? 

WDYT?

Reply:
:+1: for `path_labels`